uwblueprint/jump-math
# fix: remove header from test session drilldown pages

## Notion ticket link

[Remove Navbar from Student Results page](https://www.notion.so/uwblueprintexecs/Remove-Navbar-from-Student-Results-page-ecf1ad2e7eb14a1ca0f812f141fe01b6)



## Implementation description
- Remove header from drilldown page
- Adjust padding on drilldown page to account for changes

<img width="1438" alt="image" src="https://github.com/uwblueprint/jump-math/assets/9922514/24faf214-ebd1-43f0-8d92-8a17bee4e94a">




## Steps to test
1. View drilldown page (log in as teacher -> Assessments -> click a past assessment).
2. Verify the header does not appear and the padding looks appropriate.



## What should reviewers focus on?
* 


## Checklist
